components/micro-gateway-core/src/main/ballerina/src/gateway/constants/constants.bal

```diff
@@ -262,14 +261,9 @@ public const string JMS_CONNECTION_PASSWORD = "jmsConnectionPassword";
 public const string THROTTLE_ENDPOINT_URL = "throttleEndpointUrl";
 public const string THROTTLE_ENDPOINT_BASE64_HEADER = "throttleEndpointbase64Header";
 
-public const string TOKEN_REVOCATION_CONF_INSTANCE_ID = "tokenRevocationConfig";
```

Comment:
I hope these are unused or duplicate constants

Reply:
Most of them are duplicated. So I removed them.